projects/scapy/pcap_fuzzer.py

```diff
@@ -21,6 +21,7 @@
   import io
   import scapy
   import scapy.error
+  import scapy.layers.all
```

Comment:
Good catch, you're right, without that everything is `Raw` and the packet dissection is not tested.

Reply:
> you can add guillaume@valadon.net to receive the reports

That address was added and then replaced in https://github.com/google/oss-fuzz/pull/4940 due to https://google.github.io/oss-fuzz/faq/#why-do-you-require-a-google-account-for-authentication I think.

> I'd like to request to add pierre@droids-corp.org

I can add it but usually only gmail addresses fully work.